stats/test/handler.test.js

```diff
@@ -26,7 +26,8 @@ describe('HTTP request handler', () => {
     const handler = createHandler({
       pgPools: {
         stats: null,
-        evaluate: pgPool
+        evaluate: pgPool,
+        end: async () => {}
```

Comment:
Yeah, I didn't want to add a dummy `end` implementation in the tests, so I introduced `EndablePgPools`.

I think in this place, the `end` method should call `pgPool.end()`, shouldn't it?

```diff
- end: async () => {}
+ end: () => pgPool.end()
```

Reply:
it doesn't make a difference, the tests still finish. But I could add it for completeness?

Reply:
Either way is fine with me.